.travis/stp.sh

```diff
@@ -33,6 +33,23 @@ if [ "x${STP_VERSION}" != "x" ]; then
   sudo make install
   cd ../../
 
+  # Determine STP build flags
```

Comment:
Sorry, aligned in the semantic sense:

-DENABLE_PYTHON_INTERFACE:BOOL=OFF --> -DENABLE_PYTHON_INTERFACE:BOOL=OFF

-DONLY_SIMPLE:BOOL=ON --> -DNO_BOOST:BOOL=ON

-DSTATICCOMPILE:BOOL=ON --> -DBUILD_SHARED_LIBS:BOOL=OFF